pkg/placement/what-resolver.go

```diff
@@ -823,10 +830,6 @@ var GRsNaturedInBoth = NewMapSet(
 	mkgr("", "namespaces"),
 )
 
-var NaturedInCenterGoToMailbox = NewMapSet(
-	mkgr("apis.kcp.io", "apibindings"),
```

Comment:
https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/outline/#needs-to-be-natured-in-center-not-destined-for-edge says APIServiceBinding should be here.